src/main/resources/database/changes/release-11/add_job_completed_column.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE sample.collectionexercisejob ADD COLUMN jobcomplete boolean;
```

Comment:
It might be worth following up with the change to set the nulls to true as like @PurpleBooth mentioned there could be an issue if a collection exercise was executed at the time of deployment but this wouldn't be an issue if we made the column not null and updated any null values to true after this change had gone in

Reply:
if there's a job in-progress when we do the deployment it would need to be restarted anyway

Reply:
But wouldn't it be marked as true so won't be processed?

Reply:
sure... but there's no degradation. It's a slightly tricky edge case. I don't see the benefit of setting the old rows to true after the change has gone in. The only way to be sure is to check the created timestamps in the job table after the process has been shut down, and to flip any jobs which hadn't finished to false once the upgrade is done